f3/granite.go

```diff
@@ -460,6 +477,25 @@ func (i *instance) tryCommit(round uint32) {
 	}
 }
 
+func (i *instance) tryDecide(value ECChain) {
```

Comment:
As noted above, this method is coupling the state machine with amplification. Please separate them so this method can follow the same pattern as all the other state transitions. It should have no parameter, but be a pure function of state.

Reply:
Done.